Resources/Textures/DeltaV/Clothing/Shoes/Boots/magboots-security.rsi/meta.json

```diff
@@ -0,0 +1,41 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/7e4e9d432d88981fb9bb463970c5b98ce85c0abe",

```

Comment:
TGStation does not have Security Magboots; nor are they in this commit.

Reply:
the original sprite is theirs, I just colored over it. I guess fair use applies?

Reply:
> the original sprite is theirs, I just colored over it. I guess fair use applies?

If that's the case, add that to the attribution. "Original sprite from _____, edited by ______"